corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -17,48 +17,56 @@
       </div>
     </div>
 
-    <% if (useGrid) { %>
-    <section class="js-case-container clearfix list-cell-container-style"></section>
-    <% } else { %>
-    <div class="module-case-list-table-container">
-      <table class="table module-table module-table-case-list">
-        <% if(!useTiles) { %>
-        <thead>
-        <tr>
-          <% if (isMultiSelect && !hasNoItems) { %>
-          <th class="module-case-list-header"><input type="checkbox" id="select-all-checkbox"/></th>
-          <% } %>
-          <% _.each(headers, function(header, index) { %>
-          <% if (columnVisible(index)) { %>
-          <% if (columnSortable(index)) { %>
-          <th class="module-case-list-header header-clickable formplayer-request" data-id="<%- index %>"> <%- header %></th>
-          <% } else { %>
-          <th class="module-case-list-header" data-id="<%- index %>"> <%- header %></th>
-          <% } %>
-          <% } %>
-          <% }); %>
-        </tr>
-        </thead>
-        <% } %>
-        <% if (hasNoItems) { %>
-        <tbody>
-        <tr>
-          <td class="module-case-list-column module-case-list-column-empty" colspan="<%-headers.length%>">
-            <% if (noItemsText === null) {%>
-              <div class="alert alert-info">{% trans "List is empty." %}</div>
-            <% } else { %>
-              <div class="alert alert-info"><%- noItemsText %></div>
+    <div style="<%- columnStyle %>">
```

Comment:
Is the markup matched up correctly? I count 3 new opening div tags but 4 new closing div tags.